seerep-hdf5/seerep-hdf5-core/include/seerep-hdf5-core/hdf5-core-general.h

```diff
@@ -202,6 +216,11 @@ class Hdf5CoreGeneral
 
   inline static const std::string PROJECTNAME = "projectname";
   inline static const std::string PROJECTFRAMEID = "projectframeid";
+  inline static const std::string GEOLOC_COORDINATE_SYSTEM = "geoloc_coordinatesystem";
+  inline static const std::string GEOLOC_ELLIPSOID = "geoloc_ellipsoid";
+  inline static const std::string GEOLOC_LATITUDE = "geoloc_latitude";
+  inline static const std::string GEOLOC_LONGITUDE = "geoloc_longitude";
+  inline static const std::string GEOLOC_ALTITUDE = "geoloc_altitude";
```

Comment:
Is this a rebase error?

Reply:
i thinks so. im gonna change it right away :)